services/ui-src/src/verbiage/forms/adminBannerForm.ts

```diff
@@ -75,15 +77,16 @@ export const form = {
         label: "End date",
       },
       validation: {
-        type: "number",
+        type: "string",
         options: {
           required: true,
+          regex: /^(0[1-9]|1[0-2])\/(0[1-9]|1\d|2\d|3[01])\/(19|20)\d{2}$/,
           // TODO: add minimum start date reference min: "startDate"
         },
         errorMessages: {
           required: "End date is required",
-          format: "Invalid end date",
-          min: "End date cannot be before start date",
+          regex:
+            "Please ensure your format is in MM/DD/YYYY and is a real date",
         },
       },
```

Comment:
We'll pull all of this out to the new `abf.schema.ts` file, and can add the regex validation easily there.